src/Compilers/CSharp/Portable/Binder/Binder_Operators.cs

```diff
@@ -4377,13 +4377,13 @@ private void ValidateRefConditionalOperator(SyntaxNode node, BoundExpression tru
             var currentScope = _localScopeDepth;
 
             // val-escape must agree on both branches.
-            uint whenTrueEscape = GetValEscape(trueExpr, currentScope);
-            uint whenFalseEscape = GetValEscape(falseExpr, currentScope);
+            SafeContext whenTrueEscape = GetValEscape(trueExpr, currentScope);
+            SafeContext whenFalseEscape = GetValEscape(falseExpr, currentScope);
 
-            if (whenTrueEscape != whenFalseEscape)
+            if (!whenTrueEscape.Equals(whenFalseEscape))
```

Comment:
```suggestion
            if (whenTrueEscape != whenFalseEscape)
```

Given that we have operators here seems like we should use them.